### ISSUE TITLE
Focus color contrast ratio for non text element is less than 3:1.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#99244]
Environment Details:
VSCode Version : 1.46.0 Insider
OS Version : Win10
Additional Details:
Screenreader: NVDA[2019.3.1], JAWS[2020.70]

GitHubTags:#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.4.11;#Win10;#Visual Studio Code;#CAITestPass_May2020;#FTP;

Repro Steps:
1.Open Visual studio code insider editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3)Using  "Ctrl+\" key to split the terminal
4)Navigate to Spilt terminal icon using tab and turn on color contrast analyzer. 

Steps to Reproduce:

Actual:
Contrast ratio of focus line when it is on Spilt screen icon is less that 3:1. 

Expected:
Color contrast ration of non text contrast content should be more that 3:1 

Attachment For Reference:
![99244_A11y_VSCode_Integrated Terminal_Split terminal_Keyboard Color contrast ratio for non text element is less than 31](https://user-images.githubusercontent.com/52035395/83631614-67f81380-a5bb-11ea-9b71-dd6d3e7e4fc8.jpg)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Tyriar** [comment]
If this is a requirement for this non-text part, then it doesn't just affect the panel actions:

![image](https://user-images.githubusercontent.com/2193314/83643246-a4e4eb80-a564-11ea-848b-9e586f480fb1.png)

1.81:1:

![image](https://user-images.githubusercontent.com/2193314/83643370-cb0a8b80-a564-11ea-8506-b766233eeaab.png)

**Comment by isidorn** [comment]
This is regarding color contrast of icons.
@misolori and me are having a disucssion and we will decide what to do here.

**Comment by misolori** [comment]
I think in this case it’s referring to the focus border color.

**Comment by isidorn** [comment]
You are correct. Renamed the issue.

**Comment by misolori** [comment]
Updated this issue to reflect all focus treatments in all areas (from #99426) like Activity Bar, Sidebar, Tabs, etc.

**Comment by misolori** [comment]
@isidorn in order for us to properly fix this issue, we'll need to make sure that the focus border color passes the 3:1 ratio on our lightest background (in this case it's the activity bar). After playing with the blue color, it appears that `#007FD4` is the one that will pass the contrast ratio, but that color is significantly brighter than before (which personally I am fine with).

<img width="50%" src="https://user-images.githubusercontent.com/35271042/84544990-3d8d1100-acb3-11ea-9c33-c01da808074d.png">

![image](https://user-images.githubusercontent.com/35271042/84545139-88a72400-acb3-11ea-8ec8-5da5524d7dc5.png)

![Kapture 2020-06-12 at 13 50 22](https://user-images.githubusercontent.com/35271042/84545247-b8eec280-acb3-11ea-9b39-976ae7c22a2e.gif)

**Comment by isidorn** [comment]
@misolori thanks for looking into this and for finding a good color. I think the focus being more visible is perfectly fine, and the new color still blends well with everything. I personally prefer it to the previous one, simply because it makes the focus more clear.
So feel free to push this, especialy since this is a start of the iteration so we get feedback.

fyi @bpasero

**Comment by misolori** [comment]
Will add this today and we can gauge the feedback. This also applies to our Light theme, so will update that color to `#0090F1`

<img width="50%" src="https://user-images.githubusercontent.com/35271042/84682264-3ef44e80-aeea-11ea-9b95-0bcd2a7f47e7.png">

![Kapture 2020-06-15 at 9 28 04](https://user-images.githubusercontent.com/35271042/84682503-92ff3300-aeea-11ea-9458-c5d9f5908774.gif)

**Comment by isidorn** [comment]
Great, also works for me in Light.
